zkevm-circuits/src/super_circuit.rs

```diff
@@ -336,6 +341,67 @@ impl<const MAX_TXS: usize, const MAX_CALLDATA: usize> SuperCircuit<Fr, MAX_TXS,
         };
         Ok((k, circuit, instance))
     }
+
+    /// From the witness data, generate a SuperCircuit instance with all of the
+    /// sub-circuits filled with their corresponding witnesses.
+    ///
+    /// Also, return with it the minimum required SRS degree for the circuit and
+    /// the Public Inputs needed.
+    pub fn build_with_builder(
```

Comment:
This `build_with_builder` method has different input arguments compared to the `build` method above. Integration tests builder and eth_block are generated together with other tests.

We should combine `build_with_builder` method and `build` method together.

Reply:
Can it be a thin wrapper to avoid code copy? looks like most code are dup.